api_designer/src/main/resources/api.raml

```diff
@@ -254,6 +254,10 @@ traits:
         post:
           description: Publish project to Exchange
           is: [ commonHeaders ]
+          headers:
+            x-async-publication:
+              description: determines if publication to exchange is done sync or async
+              type: boolean
```

Comment:
I would add, just to be clear, the following attributes:
```
required: false
default: false
```
Is sync by default, right?

BTW, this is the spec:
https://github.com/raml-org/raml-spec/blob/master/versions/raml-08/raml-08.md#required
https://github.com/raml-org/raml-spec/blob/master/versions/raml-08/raml-08.md#default